model/src/main/proto/spine_examples/shareaware/market/commands.proto

```diff
@@ -52,3 +52,19 @@ message ObtainShares {
     // The quantity of the shares that the purchase process wants to obtain.
     int32 quantity = 4 [(required) = true];
 }
+
+// A command to vend shares on the market.
+message VendSharesOnMarket {
```

Comment:
Why isn't it `SellSharesOnMarket`?

And one more hint. Do we ever say "to vend shares"? Personally, I don't.